serverless/serverless-kn-source.adoc

```diff
@@ -0,0 +1,45 @@
+[id="serverless-kn-source"]
+= Using Knative Eventing sources
+include::modules/common-attributes.adoc[]
+include::modules/serverless-document-attributes.adoc[]
+:context: serverless-kn-source
+
+You can use `kn` to list available event sources or event source types for use with Knative Eventing.
+
+For information about installing `kn`, see xref:../knative-client.adoc#knative-client[Installing Knative Client].
+
+[id="serverless-list-source-types_{context}"]
+== Listing available event source types using `kn`
+
+You can list the available event source types in the terminal by using the following command:
+----
+$ kn source list-types
+----
+
+It is also possible to list available event source types in JSON format:
+
+----
+kn source list-types -o yaml
+----
+
+Currently, Knative Eventing supports the following event source types:
+
+* API server event source
+* Sink binding event source
+
+// additional modules TBA for creating and managing these sources
+
+[id="serverless-untyped-sources_{context}"]
+== Using untyped event sources
+
+Untyped sources are event sources that do not have built-in support or plugin support available by default when using `kn`. Instead, server side Custom Resource Definitions (CRDs) are evaluated to provide details about customized available event sources.
+
+You can use `kn source list-types` to find out which source types are available in your deployment.
```

Comment:
`kn source list-types` is not only for untyped sources but for all kind of sources (built-in, plugin, untyped), so its probably not correct to put it into this section. I would put it into an own, dedicated section. 

I will provide an example tomorrow. which could be added here.

Reply:
It already has its own section, it was just mentioned here too since you need to use it to find out available source types, but the untyped section is removed for now :slightly_smiling_face: